tlatools/org.lamport.tlatools/src/tlc2/tool/distributed/RMIFilenameToStreamResolver.java

```diff
@@ -138,4 +138,8 @@ public String getFullPath() {
 		}
 		return buf.toString();
 	}
+
+	public String getLibraryPath(String name) {
+		return "";
```

Comment:
What was the old behavior of RMIFilenameToStreamResolver WRT libraries?  Perhaps, throw an UnsupportedOperationExp here?

Reply:
RMIFilenameToStreamResolver is used in the scope of distributed TLC. A user does not have to copy the spec files to the worker/compute nodes.  Instead, compute nodes read the modules through the RMIFilenameToStreamResolver.  I don't remember if RMIFilenameToStreamResolver ever supported additional librarypath locations. If it did, please check that it still.

Reply:
https://tla.msr-inria.inria.fr/tlatoolbox/doc/model/distributed-mode.html

Reply:
As we are not modifying the interface anymore, I guess we don't need to worry about RMI and RCP now o/